bootstrap/src/Main.hs

```diff
@@ -93,7 +93,7 @@ main2 meta plan = do
                         return
                             ( Hackage
                             , Just $ fromIntegral (I.riRevision relInfo)
-                            , P.sha256FromByteString $ I.getSHA256 $ I.riCabal relInfo
+                            , P.sha256FromByteString $ I.getSHA256 $ I.riCabalHash relInfo
```

Comment:
This probably needs some sort of `#ifdef` in order to not break bootstrapping with GHC `<9.4.0`.

Reply:
@nmeum: thank you! Yes, I guess it does. We'll need to test this with older GHC and tweak as needed.

Reply:
It seems this change was introduced in https://github.com/haskell-CI/haskell-ci/commit/4c28d9961d5d3e69c84205693824cd62fb9a7672 with cabal-install-parsers-0.6. I added a corresponding `#if MIN_VERSION` check.